cpu/esp32/esp-eth/esp_eth_netdev.c

```diff
@@ -109,11 +109,14 @@ static esp_err_t IRAM_ATTR _eth_input_callback(void *buffer, uint16_t len, void
     return ESP_OK;
 }
 
-#if EMAC_PHY_POWER_PIN != GPIO_UNDEF
 static void _esp_eth_phy_power_enable_gpio(bool enable)
 {
     assert(EMAC_ETHERNET_PHY_CONFIG.phy_power_enable);
 
+    if (EMAC_PHY_POWER_PIN == GPIO_UNDEF) {
```

Comment:
This seems unrelated to this PR, or maybe I'm missing something ?

Reply:
It is related since I had to change with the removing the the GPIO_UNDEF macro
```
#define GPIO_UNDEF (0xff)
```
and using the default definition from `periph/gpio.h`.
```
#define GPIO_UNDEF          ((gpio_t)(UINT_MAX))
```
Since the latter one uses a type, I could not use `GPIO_UNDEF` anymore to disable the whole function. Now, the function exists but returns.
```diff
- #if EMAC_PHY_POWER_PIN != GPIO_UNDEF	
  static void _esp_eth_phy_power_enable_gpio(bool enable)
  {
      assert(EMAC_ETHERNET_PHY_CONFIG.phy_power_enable);
+     if (EMAC_PHY_POWER_PIN == GPIO_UNDEF) {
+        return;
+    }
      ...
  }
- #endif